Formula/w/whisper-cpp.rb

```diff
@@ -0,0 +1,20 @@
+require "English"
+class WhisperCpp < Formula
+  desc "Port of OpenAI's Whisper model in C/C++"
+  homepage "https://github.com/ggerganov/whisper.cpp"
+  url "https://github.com/ggerganov/whisper.cpp/archive/refs/tags/v1.4.0.tar.gz"
+  sha256 "b2e34e65777033584fa6769a366cdb0228bc5c7da81e58a5e8dc0ce94d0fb54e"
+  license "MIT"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make"
+    bin.install "main" => "whisper-cpp"
+  end
```

Comment:
or you can do this

```
  depends_on "cmake" => :build

  def install
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
    bin.install "bin/main" => "whisper-cpp"
  end
```

This would provide shared lib and header file

```
├── bin
│   └── whisper-cpp
├── include
│   └── whisper.h
└── lib
    └── libwhisper.dylib
```